ballerina/client.bal

```diff
@@ -1,4 +1,7 @@
-// Copyright (c) 2024, WSO2 LLC. (http://www.wso2.com).
+// AUTO-GENERATED FILE. DO NOT MODIFY.
+// This file is auto-generated by the Ballerina OpenAPI tool.
+
+// Copyright (c) {{LICENSE_YEAR}}, WSO2 LLC. (http://www.wso2.com).
```

Comment:
License should be as follows.
```suggestion
// Copyright (c) 2024, WSO2 LLC. (http://www.wso2.com).
```

Please update the license year in the `license.txt` file and try regenerating the client